Sources/GroundState/vmc_sampling.cc

```diff
@@ -0,0 +1,165 @@
+#include "vmc_sampling.hpp"
+
+namespace netket {
+namespace vmc {
+
+Result ComputeSamples(AbstractSampler &sampler, Index nsamples,
+                      Index ndiscard) {
```

Comment:
I think we should an option to compute or not compute the log derivatives in this loop (by default I would say we shouldn't, and VmcResult should make sure they are computed if one asks for them later on?)

Reply:
There is now an option to disable computation of the log-derivatives.